gbfs.md

```diff
@@ -395,6 +395,7 @@ Field Name | REQUIRED | Type | Defines
 \- `propulsion_type` | Yes | Enum | The primary propulsion type of the vehicle. <br /><br />Current valid values are:<br /><ul><li>`human` _(Pedal or foot propulsion)_</li><li>`electric_assist` _(Provides power only alongside human propulsion)_</li><li>`electric` _(Contains throttle mode with a battery-powered motor)_</li><li>`combustion` _(Contains throttle mode with a gas engine-powered motor)_</li></ul> This field was inspired by, but differs from the propulsion types field described in the [Open Mobility Foundation Mobility Data Specification](https://github.com/openmobilityfoundation/mobility-data-specification/blob/master/provider/README.md#propulsion-types).
 \- `max_range_meters` | Conditionally REQUIRED | Non-negative float | If the vehicle has a motor (as indicated by having a value other than `human` in the `propulsion_type` field), this field is REQUIRED. This represents the furthest distance in meters that the vehicle can travel without recharging or refueling when it has the maximum amount of energy potential (for example, a full battery or full tank of gas).
 \- `name` | OPTIONAL | String | The public name of this vehicle type.
+\- `return_type` | OPTIONAL | Array | The conditions for returning the vehicle at the end of the trip. For vehicles that have more than one return option, include all applicable methods in the array. <br /><br />Current valid values are:<br /><ul><li>`free_floating` _(The vehicle can be returned anywhere permitted within the service area)_</li><li>`roundtrip` _(The vehicle must be returned to the initial rental station)_</li><li>`station` _(The vehicle must be returned to any station within the service area - note that a specific station can be defined in [free_bike_status.json](#free_bike_status.json) using `home_station`)_
```

Comment:
Hello! Could you add:
- That free_floating vehicles return is subject to geofencing zones if such feed is defined
- That rountrip value cannot be used in combination with free_floating bikes. It doesn't make sense.
- I would rename station to any_station, and rountrip to roundtrip_station. And would add any_station cannot be used in combination with roundrtip_station (the two don't make sense together). Renaming them makes it clear that the two are exclusive.

I also left a comment on the proposal doc about home_station systems. I would like to see some more examples of such system.

Reply:
Thanks for your comments Kristian, I will make those changes! I will answer your question about home_station in the proposal document.